src/opm/parser/eclipse/share/keywords/000_Eclipse100/F/FOAMROCK

```diff
@@ -0,0 +1,6 @@
+{"name" : "FOAMROCK" , "sections" : ["SPECIAL", "PROPS"], "size" : {"keyword":"TABDIMS" , "item":"NTSFUN"}, "items":
+    [   {"name" : "ADSORPTION_INDEX", "value_type" : "INT", "default" : 1},
```

Comment:
This looks weird - the integer index `ADSORPTION_INDEX` is converted to an enum with names `Yes` and `No` - what happened?

Reply:
Calling the thing ADSORPTION_INDEX in the first place is the weird thing, but that is not terminology I invented... It is 1 if desorption is allowed (and immediate) and 2 if not. In any case it will not exist anymore as I change to a bool...